apps/lend/src/layout/Header.tsx

```diff
@@ -32,7 +32,7 @@ const Header = ({ chainId, sections }: HeaderProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [footerHeight])
 
-  const { rLocalePathname } = getParamsFromUrl()
+  const { rLocalePathname, rNetwork } = getParamsFromUrl()
```

Comment:
You're using `getNetworkFromUrl()` in `dao` and `loan` apps, but `getParamsFromUrl()` in `lend` and `main` apps. I was expecting it to be the same for all 4. Probably still works and might be something to fix once we harmonize all apps into a single layout.

EDIT: ah nvm I see `getParamsFromUrl()` calls `getNetworkFromUrl()` and some more.